shell/browser/electron_browser_client.h

```diff
@@ -338,6 +361,9 @@ class ElectronBrowserClient : public content::ContentBrowserClient,
 #if BUILDFLAG(IS_MAC)
   ElectronBrowserMainParts* browser_main_parts_ = nullptr;
 #endif
+  // SAP-15679: Notification doesn't work when using Web URL in a restricted
+  // container
+  content::BrowserContext* notifications_browser_context_;
```

Comment:
I think this code is getting deleted per my review above, but if it somehow sticks this should be a `raw_ptr<T>` not `T*`

Reply:
`I think this code is getting deleted per my review above, but if it somehow sticks this should be a raw_ptr<T> not T*` even above we have raw pointer. We do not need to worry about life circle of these pointer it is longer than time of access to it

Reply:
please consider my remark about set_notify_context